integration-tests/test/rnode.py

```diff
@@ -523,10 +524,10 @@ def make_bootstrap_node(
     ])
 
     container_command_options = {
-        "--port":                   40400,
-        "--validator-private-key":  keypair.private_key,
-        "--validator-public-key":   keypair.public_key,
-        "--host":                   container_name,
```

Comment:
This tool is making the code less readable.  Is there a way to disable particular formatting hint?

Reply:
https://www.python.org/dev/peps/pep-0008/#pet-peeves that's pep8 suggest.